### ISSUE TITLE
Tags fail to get sent with new droplets

### ISSUE THREAD

**Issue by caseyhen** [koalalorenzo/python-digitalocean#178]
When creating a new droplet and passing through an array of tag, they do not get sent along when the droplet is sent to the api.

```
droplet = digitalocean.Droplet(
      token="XXXXXXX",
      name='Example',
      region='nyc3', # New York 3
      image='ubuntu-16-04-x64', # Ubuntu 16.04.1 x64
      size_slug='512mb',  # 512MB
      backups=False,
      private_networking=True,
      tags=['tag1','tag2'])

droplet.create()
```

**Comment by andrewsomething** [comment]
:+1:  This was added after the initial tagging API was introduced. Looks like never made it in here.

https://developers.digitalocean.com/documentation/changelog/api-v2/tagging-in-droplet-create/

**Comment by koalalorenzo** [comment]
Yes, at the moment it is not supported. @caseyhen if you want to contribute to implement it, you are more than welcome.

**Comment by koalalorenzo** [comment]
Closed in #179
